Launcher/MainWindow.xaml.cs

```diff
@@ -826,7 +794,7 @@ private void browse_model_Click(object sender, RoutedEventArgs e)
 
             if (dlg.ShowDialog() == CommonFileDialogResult.Ok)
             {
-                compile_model_path.Text = dlg.FileName.Replace(H2Ek_install_path + "data\\", "");
+                compile_model_path.Text = dlg.FileName;
```

Comment:
Is this an issue with anything else, I think we I wrote a lot of this code I thought you couldn't give the full path.

Reply:
As far as I could tell from testing it myself the only thing that didn't accept the full path was the DAEConverter. The DAEConverter wouldn't compile the object but the regular toolset accepts this fine. This appears to be because the DAEConverter is adding the path of the exe to the path the user inputs for compiling the object.